Automattic/jetpack
# Paywall: Switching accounts URL fix

Fixes https://github.com/Automattic/jetpack/issues/36307

## Proposed changes:

It:
- changes the switching account link to be sentence-cased to be consistent with e.g. "Already a subscriber?"
- logs in the user using the login link flow when clicking the "Switch accounts" link (consistent with the Subscriber Login block flow)

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






1. Be logged in to wp.com
2. Open a post with a Paywall block on a Simple site that you are not subscribed to
3. Click "Switch accounts"
4. Log in to another account
5. Make sure the auth token is not visible in the URL after you were redirected to the site
